ci/ray_ci/automation/docker_tags_lib.py

```diff
@@ -13,6 +13,51 @@
 bazel_workspace_dir = os.environ.get("BUILD_WORKSPACE_DIRECTORY", "")
 SHA_LENGTH = 6
 
+PYTHON_VERSIONS_RAY = ["", "py39", "py310", "py311"]
```

Comment:
could you encode a default python version or something? rather than having `""` in the array?

Reply:
Can I use this same logic here: https://github.com/ray-project/ray/blob/master/ci/ray_ci/docker_container.py#L122?
Since py39 is the default version, we can just add `""` whenever we see py39